store.go

```diff
@@ -318,11 +331,21 @@ type Store interface {
 	ApplyDiffWithDiffer(to string, options *drivers.ApplyDiffWithDifferOpts, differ drivers.Differ) (*drivers.DriverWithDifferOutput, error)
 
 	// ApplyDiffFromStagingDirectory uses stagingDirectory to create the diff.
+	// Deprecated: it will be removed soon.
```

Comment:
```suggestion
	// Deprecated: it will be removed soon. Use CleanupStagingDirectory instead.
```
... or however it is going to be called.

Reply:
“`Use ApplyStagedLayer`…” here, my mistake.